azerothcore/azerothcore-wotlk
# fix(DB/smart_scripts): Fix Quest: Stranglethorn Fever

## Changes Proposed:
-  Update [Witch Doctor Unbagwa](https://classic.wowhead.com/npc=1449/witch-doctor-unbagwa) faction so he would aggro gorillas and die if no players present to aggro gorillas
- Thanks [natnat201193](https://github.com/natnat201193) for sniffed data
## Issues Addressed:

- Closes [Quest "Stranglethorn Fever" won't trigger the summon for "Mokk" #1065](https://github.com/chromiecraft/chromiecraft/issues/1065)
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6774
- Closes https://github.com/chromiecraft/chromiecraft/issues/1065

## SOURCE:


## Tests Performed:

- Added quest, started sequence, evaded gorillas and they killed Witch Doctor, abandoned quest. Added quest again and waited until gorillas de-spawns and Doctor respawns. Started doctor summon quest again. 
- 


## How to Test the Changes:


1. Add quest (.q add 348), add item(.add 2799 10)
2. Talk to doctor to start summoning Mokka
3. Turn on gm mode(.gm on) and let gorillas kill Doctor.
4. No abandon quest. (.q remove 348), add q again (.q add 348)
5. Wait until gorillas de-spawn and Doctor respawns (takes time )
6. Confirm u can start quest (summoning sequence) again.

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
